go.mod

```diff
@@ -37,14 +37,14 @@ require (
 	github.com/minio/minio-go/v6 v6.0.57
 	github.com/peterhellberg/duration v0.0.0-20191119133758-ec6baeebcd10
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.12.1
+	github.com/prometheus/client_golang v1.12.2
 	github.com/prometheus/client_model v0.4.0
 	github.com/robfig/cron v1.2.0
 	github.com/sirupsen/logrus v1.9.0
-	github.com/spf13/viper v1.10.1
+	github.com/spf13/viper v1.12.0
 	github.com/stretchr/testify v1.8.4
 	gocloud.dev v0.22.0
-	golang.org/x/net v0.19.0
+	golang.org/x/net v0.21.0
```

Comment:
can we not change this to v0.23.0 instead of having a replace directive [here](https://github.com/opendatahub-io/data-science-pipelines/pull/55/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R73-R74)?

Reply:
Done, updated to have this change.